### PR TITLE
add hooks for rsvp

### PR DESCRIPTION
**Pull request by 75nineteen**
Ref: #92 

/cc @borkweb @zbtirrell

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | Whitespace found at end of line | [RSVP.php:315](https://github.com/moderntribe/event-tickets/blob/5d01701850a88c62255480c98037a165a2b3f4c4/src/Tribe/RSVP.php#L315)
:x: | Whitespace found at end of line | [RSVP.php:318](https://github.com/moderntribe/event-tickets/blob/5d01701850a88c62255480c98037a165a2b3f4c4/src/Tribe/RSVP.php#L318)
:x: | Whitespace found at end of line | [RSVP.php:321](https://github.com/moderntribe/event-tickets/blob/5d01701850a88c62255480c98037a165a2b3f4c4/src/Tribe/RSVP.php#L321)
:x: | Whitespace found at end of line | [RSVP.php:323](https://github.com/moderntribe/event-tickets/blob/5d01701850a88c62255480c98037a165a2b3f4c4/src/Tribe/RSVP.php#L323)

> via jenkins/codesniffer-phpcs

File: src/Tribe/RSVP.php [hunk 1]
```diff
@@ -312,8 +312,15 @@ public function generate_tickets( ) {
 				update_post_meta( $attendee_id, $this->order_key, $order_id );
 				update_post_meta( $attendee_id, $this->full_name, $attendee_full_name );
 				update_post_meta( $attendee_id, $this->email, $attendee_email );
+				
+				do_action ( 'tickets_rsvp_tickets_created', $attendee_id, $event_id, $product_id );
```
Comment (borkweb) [inline src/Tribe/RSVP.php]: Can this action be changed to: `event_tickets_rsvp_ticket_created`?

File: src/Tribe/RSVP.php [hunk 2]
```diff
@@ -312,8 +312,15 @@ public function generate_tickets( ) {
...
 			}
+			
+			do_action( 'tickets_rsvp_tickets_generated_for_product', $product_id, $order_id, $qty );
```
Comment (borkweb) [inline src/Tribe/RSVP.php]: Can this action be changed to: `event_tickets_rsvp_tickets_generated_for_product`?

File: src/Tribe/RSVP.php [hunk 3]
```diff
@@ -312,8 +312,15 @@ public function generate_tickets( ) {
...
 		}
+		
+		do_action ( 'tickets_rsvp_tickets_generated', $order_id );
```
Comment (borkweb) [inline src/Tribe/RSVP.php]: Can this action be changed to: `event_tickets_rsvp_tickets_generated`?

**Comment by borkweb** [comment]
Thank you for this Pull Request! Those actions are a good idea :D

**Comment by 75nineteen** [comment]
@borkweb Absolutely. Just pushed over an update.

**Comment by borkweb** [comment]
Awesome! Thanks :) I'll merge this and make a tweak to that first action to make the second `tickets` singular: `event_tickets_rsvp_ticket_created`

**Pull request merged**